# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1495582141 25200
#      Tue May 23 16:29:01 2017 -0700
# Branch stable
# Node ID 22ed4204b2b793dbc43b0bb5b00c5b93d45020b7
# Parent  22fd2ad85218c2af1942e9c72682faf3ae42f7bd
maint: fix file permissions on source files

* TerminalView.cpp, TerminalView.h, pkg.m: Fix permissions, unset the
executable bit.

diff --git a/libgui/qterminal/libqterminal/unix/TerminalView.cpp b/libgui/qterminal/libqterminal/unix/TerminalView.cpp
old mode 100755
new mode 100644
diff --git a/libgui/qterminal/libqterminal/unix/TerminalView.h b/libgui/qterminal/libqterminal/unix/TerminalView.h
old mode 100755
new mode 100644
diff --git a/scripts/pkg/pkg.m b/scripts/pkg/pkg.m
old mode 100755
new mode 100644
